TodoApp/TodoApp.Api.Tests/Controllers/ItemsControllerTests.cs

```diff
@@ -62,65 +61,61 @@ public void TearDown()
         public async Task GetReturnsAllItems()
         {
             var actionResult = await _controller.GetAsync();
-
             var contentResult = await actionResult.ExecuteAsync(CancellationToken.None);
             contentResult.TryGetContentValue(out Item[] value);
 
             Assert.That(contentResult.Content, Is.Not.Null);
             Assert.That(contentResult.StatusCode, Is.EqualTo(HttpStatusCode.OK));
-            Assert.That(value, Is.EqualTo(_items).AsCollection.UsingItemEqualityComparer());
+            Assert.That(value, Is.EqualTo(_items).AsCollection.UsingItemEqualityComparer(),
+                "Collections are not equal");
         }
 
         [Test]
         public async Task GetReturnsItemWithSameId()
         {
             var actionResult = await _controller.GetAsync(Guid1);
-
             var contentResult = await actionResult.ExecuteAsync(CancellationToken.None);
             contentResult.TryGetContentValue(out Item item);
 
             Assert.That(contentResult.Content, Is.Not.Null);
             Assert.That(contentResult.StatusCode, Is.EqualTo(HttpStatusCode.OK));
-            Assert.That(item, Is.EqualTo(_items[0]).UsingItemEqualityComparer());
+            Assert.That(item, Is.EqualTo(_items[0]).UsingItemEqualityComparer(), "Items are not equal");
         }
 
         [Test]
         public async Task PostSetsLocationHeader()
         {
-            var actionResult =
-                await _controller.PostAsync(
-                    new Item("updatedText", new Guid("e6eb4638-38a4-49ac-8aaf-878684397705")));
+            var itemToPost = new Item {Text = "updatedText", Id = Guid1};
```

Comment:
That is the relation between `itemToPost` and `ItemToPost`? It is a bit misleading now, since `Guid1` is different than the GUID in response and posted item. You want the test to work for dummy controller as well for the real one, to express and define a (happy/valid in this very case) path through Your application...